### PR TITLE
[WIP] Create Release notes PR for v0.5.3

### PR DESCRIPTION
**Pull request by ramrodo**
**Description**

This PR creates Release notes PR for v0.5.3

**Notes for Reviewers**

**NOT MERGE**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

File: .github/workflows/release-notes.yml [hunk 1]
```diff
@@ -1,46 +1,18 @@
...
-            printf '%b\n' "---\nlayout: release\ndate: $(date +'%Y-%m-%d')\ntag: ${{ steps.release_drafter.outputs.tag_name }}\n---\n\n${{ steps.release_drafter.outputs.body }} " > ./docs/_releases/${{ steps.release_drafter.outputs.tag_name }}.md
-
   create_release_notes_pr:
     runs-on: ubuntu-latest
     steps:
+      - name: Checkout
+        uses: actions/checkout@v2
       - name: Create Pull Request
         id: cpr
         uses: peter-evans/create-pull-request@v3
```
Comment (leecalcote) [inline .github/workflows/release-notes.yml:13]: Is the move from v5 to v3 intentional?

Comment (ramrodo) [reply .github/workflows/release-notes.yml:13]: No, this was only for testing. I'll use the right one.

Comment (leecalcote) [reply .github/workflows/release-notes.yml:13]: Oooh, ok. 🙂

**Comment by ramrodo** [comment]
Closing in favor of https://github.com/layer5io/meshery/pull/2811

**Comment by ramrodo** [comment]
Re-opening for doing the last test...

**Comment by ramrodo** [comment]
It seems this PR will not work because it's a PR. I need to test it directly at creating a new release.

**Comment by ramrodo** [comment]
Try one more time

**Comment by ramrodo** [comment]
It seems I should create a branch, not from a fork...

**Pull request closed without merging**